scripts/linters/js_ts_linter.py

```diff
@@ -927,3 +926,26 @@ def perform_all_lint_checks(self):
             return []
 
         return self._lint_js_and_ts_files()
+
+
+def get_linters(js_filepaths, ts_filepaths, verbose_mode_enabled=False):
+    """Creates JsTsLintChecksManager and ThirdPartyJsTsLintChecksManager
+        objects and return them.
+
+    Args:
+        js_filepaths: list(str). A list of js filepaths to lint.
+        ts_filepaths: list(str). A list of ts filepaths to lint.
+        verbose_mode_enabled: bool. True if verbose mode is enabled.
+
+    Returns:
+        linter: object(linter). Returns linter object.
+    """
+    js_ts_file_paths = js_filepaths + ts_filepaths
```

Comment:
> Because we do not have separate linters for Js and Ts files and if we use both 'js' and 'ts' together then files will be linted twice.

If the above is the case then why do we have js_filepaths and ts_filepaths?

Should we just say "ts_filepaths" and forbid linting of js? Do we still have js files in the codebase?

Reply:
We still have js files in core/tests/protractor_desktop.
We are going to need both js_filepaths and ts_filepaths. In JsTslinter js_filepaths, ts_filepaths, js_ts_filepaths all are used in different methods.